callout/src/test/java/com/google/apigee/ProtoMessageBuilders/ExecutionContextProtoMessageBuilderTest.java

```diff
@@ -0,0 +1,279 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.apigee.ProtoMessageBuilders;
+
+import com.apigee.flow.Fault;
+import com.apigee.flow.execution.ExecutionContext;
+import com.google.apigee.Execute;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.MockitoAnnotations;
+import org.mockito.Spy;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+
+public class ExecutionContextProtoMessageBuilderTest {
+
+  private final String ATTRIBUTES_KEY1 = "key1";
+  private final String ATTRIBUTES_KEY2 = "key2";
+  private final String ATTRIBUTES_VALUE1 = "value1";
+  private final String ATTRIBUTES_VALUE2 = "value2";
+  private final String FAULT_NAME = "faultname";
+  private final String FAULT_REASON = "faultreason";
+  private final String FAULT_SUB_CATEGORY = "faultsubcategory";
+  Fault mockFault1 = mock(Fault.class);
+  Fault mockFault2 = mock(Fault.class);
```

Comment:
I think you can do @Mock mockFault2;

@Mock takes effect when the following in used in @Before

    MockitoAnnotations.initMocks(this);

You call openMocks(this).  You can see how the two differ.  Maybe one covers the other.

Reply:
https://www.javadoc.io/doc/org.mockito/mockito-core/latest/org/mockito/MockitoAnnotations.html

initMocks has been deprecated and replaced with openMocks in mockito 3.4